sdk/src/Core/Amazon.Runtime/Credentials/BasicAWSCredentials.cs

```diff
@@ -38,11 +38,21 @@ public class BasicAWSCredentials : AWSCredentials
         /// </summary>
         /// <param name="accessKey"></param>
         /// <param name="secretKey"></param>
-        public BasicAWSCredentials(string accessKey, string secretKey)
+        public BasicAWSCredentials(string accessKey, string secretKey) : this (accessKey, secretKey, null)
+        {
+        }
+
+        /// <summary>
+        /// Constructs a BasicAWSCredentials object for the specified accessKey, secretKey, and accountId
+        /// </summary>
+        /// <param name="accessKey"></param>
+        /// <param name="secretKey"></param>
+        /// <param name="accountId"></param>
```

Comment:
Lets add some descriptions for this parameters. I think in the past access key and secret key were obvious but account id not so much, particular what it is used for. Also if there are format requirements like the account id with no hyphens we should call that out. Once add comments for the parameters update the other constructor with the docs for access key and secret key.